ansys/dpf/core/misc.py

```diff
@@ -122,8 +122,6 @@ def find_ansys():
     for version in sorted(versions, reverse=True):
         if not version.isnumeric():
             continue
-        if version == __ansys_version__:
```

Comment:
@cbellot000 that means that in __ansys_version__ = 222 case, find_ansys() method won't be able to return an ansys_path based on AWP_ROOT222 ?

Reply:
@cbellot000 @anslpa yes the next elif might have been left by mistake. The get_ansys_path does try AWP_ROOT222 before calling find_ansys(), but you might want to still try to get it within find_ansys() for retro-compatibility.